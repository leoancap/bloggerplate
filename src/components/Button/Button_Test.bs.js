// Generated by ReScript, PLEASE EDIT WITH CARE
'use strict';

var React = require("react");
var Button = require("./Button.bs.js");
var Render = require("../../lib/Render/Render.bs.js");
var TestUtils = require("../../lib/TestUtils/TestUtils.bs.js");

var button = React.createElement(Button.make, {
      children: Render.s("ButtonText"),
      onClick: (function (param) {
          
        })
    });

TestUtils.test("Component renders", (function (param) {
        return TestUtils.toMatchSnapshot(TestUtils.expect(TestUtils.getByText({
                            NAME: "Str",
                            VAL: "ButtonText"
                          }, undefined, TestUtils.render(undefined, undefined, undefined, undefined, undefined, button))));
      }));

exports.button = button;
/* button Not a pure module */
