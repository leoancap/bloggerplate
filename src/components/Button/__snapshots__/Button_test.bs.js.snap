// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders 1`] = `
<button
  class="css-1ehxhbq css-0"
>
  ButtonText
</button>
`;
